Urban-Fun/View Controllers/SearchViewController.m

```diff
@@ -263,6 +273,10 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         vc.filtersDictionary = self.filtersDictionary;
         vc.filtersVCDelegate = self;
     }
+    if ([[segue identifier] isEqualToString:@"googleMaps"]){
```

Comment:
instead of hardcoding the segue identifiers, it would be better to create helper class and create some constants for them. That way you don't have to retype the name each time with a risk of typo, and you get the benefit of autocomplete with Xcode.

Reply:
On top of that, I'd suggest to make this identifiers more easy to understand, if you're going to GoogleMapsViewController i'd name this as `GoogleMapsViewControllerSegueIdentifier`